### PR TITLE
Add forking info action

### PR DESCRIPTION
**Pull request by bthaile**
add pop forking information

update the UI to use new ForkingInfo type

image

### CODE REVIEW

File: packages/augur-ui/src/modules/universe/actions/load-forking-info.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+) {
+  return async (
+    dispatch: ThunkDispatch<void, any, Action>,
+    getState: () => AppState
+  ) => {
+    const { universe } = getState();
+    if (universe && universe.id && universe.id !== incomingUniverse) return;
+    const forkingMarket = forkingMarketId || (await getForkingMarket());
+    const isForking =
+      forkingMarket !== '0x0000000000000000000000000000000000000000';
```
Comment (adrake33) [inline packages/augur-ui/src/modules/universe/actions/load-forking-info.ts]: Isn't there a constant for NULL_ADDRESS that can be used here?

Comment (bthaile) [reply packages/augur-ui/src/modules/universe/actions/load-forking-info.ts]: good one I'll change that

**Review by adrake33 — APPROVED** [review]
🍴

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://featurepeek.com). 🌶

 This link will always point to this PR's latest changes: https://peek.run/exohx1w

**Pull request merged**